ex2_hal/adcs/equipment_handler/test/adcs_binary_test.h

```diff
@@ -14,12 +14,16 @@
 #include "adcs_io.h"
 #include "FreeRTOS.h"
 
+// Programs List Enumeration Values
+#define BOOTLOADER 0
```

Comment:
This macro should not be given such a generic name. ADCS_BOOTLOADER would be better so that conflicts with other parts of the code are avoided